writer.go

```diff
@@ -718,6 +746,21 @@ func (p *MediaPlaylist) Close() {
 	p.Closed = true
 }
 
+// SetDefaultKey sets encryption key appeared once in header of the
+// playlist (pointer to MediaPlaylist.Key). It useful when keys not
+// changed during playback.  Set tag for the whole list.
+func (p *MediaPlaylist) SetDefaultKey(method, uri, iv, keyformat, keyformatversions string) error {
+	// A Media Playlist MUST indicate a EXT-X-VERSION of 5 or higher if it
+	// contains:
+	//   - The KEYFORMAT and KEYFORMATVERSIONS attributes of the EXT-X-KEY tag.
+	if keyformat != "" || keyformatversions != "" {
```

Comment:
The spec says "and" but perhaps that is not important?

Reply:
This is grafov-code and  just a revert of my earlier deletion.